controllers/homeRoutes.js

```diff
@@ -114,12 +114,20 @@ router.get("/", async (req, res) => {
     const categoryData = await Category.findAll({
       include: [{ model: Product, include: [{ model: User }] }],
     });
+
     const allCategories = categoryData.map((category) =>
       category.get({ plain: true })
     );
     console.log(allCategories);
+
     // Pass serialized session value into homepage template
-    res.render('homepage',{ ...dataObject, logged_in: req.session.logged_in});
+    res.render("homepage", {
+      logged_in: req.session.logged_in,
+      // Mock shelves can be removed when the backend feature is ready
+      shelves,
```

Comment:
Shelves is mock data that will need to be replaced by the backend when it's ready.